src/midgard/logging.cc

```diff
@@ -16,7 +16,7 @@
 namespace {
 
 inline std::tm* get_gmtime(const std::time_t* time, std::tm* tm) {
-#ifdef _MSC_VER
+#ifdef _WIN32
```

Comment:
I actually think I forgot to check if this is necessary for mingw

Reply:
Perhaps, either MinGW can link against msvcrt or it takes `gmtime_s` as part of C++11, https://en.cppreference.com/w/c/chrono/gmtime

Reply:
Just tried with the old _MSC_VER, but it doesn't find `gmtime_s`.

Reply:
I'd expect that with `#ifdef _MSC_VER`, the MSVC should look for `gmtime_s` and GCC should look for `gmtime_r`, with success, as here https://godbolt.org/z/vcP3vv

Question is, which one can MinGW find?

Reply:
Not sure I understand. MinGW doesn't find `gmtime_s` but apparently finds `gmtime_r` as that is what works. As I understand mingw links to msvcrt.dll.

Reply:
https://godbolt.org, sweet tool BTW :+1:

Reply:
Right, so if your change works for MinGW and MSVC, it's good then 👍